tests/components/onewire/const.py

```diff
@@ -435,6 +435,29 @@
             },
         ],
     },
+    "28.222222222222": {
+        ATTR_INJECT_READS: [
```

Comment:
Please add a comment:
```suggestion
    "28.222222222222": {
        # This device has precision options in the config entry
        ATTR_INJECT_READS: [
```

Reply:
Done